### ISSUE TITLE
Add Locale Data

### ISSUE THREAD

**Issue by AndreasPizsa** [AndreasPizsa/parse-decimal-number#1]
Get data from a standards database, then create super compact structure with gulp, like

``` coffeescript
# locales.coffee
'.,dedeDEdeAT ,fr,.enenUK -kz'.match /([^a-z])([^a-z])(?:([a-z]{2}([A-Z]{2})?)+/
```

should be <1k for all countries/languages

Then do `parseDecimalNumber('123 456 789,0','fr')`

Introduce `guessLocalesFromDecimalNumber(string)`?

**Comment by XueWei** [comment]
I read the document, it say: Numeral.js is good at formatting numbers and comes with an extensive set of locale data that you can use with parse-decimal-number.
I tried 
parseDecimalNumber('12.345.678,90', numeral.countryData('de').delimiters);

but it complain : console.log(numeral.countryData('de').delimiters);
                    ^
TypeError: numeral.countryData is not a function

Could you please let me know how to use locale format data?

**Comment by XueWei** [comment]
I did try to use localedata from https://github.com/adamwdraper/Numeral-js/tree/master/locales, but it seems some locale data are not correct, for example, de thousand separators should be ".". 
Could you support locale data which come from CLDR in this project?

**Comment by AndreasPizsa** [comment]
@XueWei Thanks for your commemnt.

I've just released version 1.0.0, which adds [support for CLDR data](https://github.com/AndreasPizsa/parse-decimal-number#using-with-cldr).

👍

**Comment by XueWei** [comment]
@AndreasPizsa  Thanks for update quickly. Yeah, that is great! I don't need call CLDR js by myself. One more question. How about the size increase if I import CLDR?  For example, if I only need number data for fr-FR, when I require 'cldr', does this import all locales' data from cldr?
